library/blockly/html/generator.js

```diff
@@ -563,6 +563,23 @@ htmlGen['summary'] = function(block){
     return code;
 };
 
+htmlGen['rgba_picker'] = function(block){
+	var r = looseEscape(block.getFieldValue('r'));
+	var g = looseEscape(block.getFieldValue('g'));
+	var b = looseEscape(block.getFieldValue('b'));
+	var a = looseEscape(block.getFieldValue('a'));
+	
+	return "rgba(" + r + ", " + g + ", " + b + ", " + a + ")";
+}
+
+htmlGen['hex_picker'] = function(block){
+	return "#" + looseEscape(block.getFieldValue('color'));
+}
+
+htmlGen['color_picker'] = function(block){
+	return looseEscape(block.getFieldValue('color'));
```

Comment:
If this is a colour picker value, why does it need any escaping? It's logically impossible for non-escaped code to be passed in here by a Blockly built-in module. Escaping makes it slower in big projects.